test/suites/integration/launchtemplates_test.go

```diff
@@ -61,6 +63,46 @@ var _ = Describe("LaunchTemplates", func() {
 
 		ExpectInstance(pod.Spec.NodeName).To(HaveField("ImageId", HaveValue(Equal(customAMI))))
 	})
+	It("should merge UserData contents for AL2 AMIFamily", func() {
+		content, _ := ioutil.ReadFile("testdata/al2_userdata_input.golden")
```

Comment:
We need an expect error here in case the file gets moved or is not found for some reason